hpi-schul-cloud/dof_app_deploy
# BC-4955 - Activate privacy policy upload on new admin page

# Description

QA note:
Change tested, no issues found. All acceptance criteria are met, but verification only by code. To be sure of the change, the change should be checked after implementation when the change is implemented from env main to env staging and env staging to env prod. see tickiet: https://ticketsystem.dbildungscloud.de/browse/BC-4955


## Links to Tickets or other pull requests
https://ticketsystem.dbildungscloud.de/browse/BC-4955
https://github.com/hpi-schul-cloud/nuxt-client/pull/2783
https://github.com/hpi-schul-cloud/schulcloud-server/pull/4375


## Changes



## Datasecurity



## Deployment



## New Repos, NPM pakages or vendor scripts



## Screenshots of UI changes



## Approval for review


> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.
